src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useAllowanceApproval.tsx

```diff
@@ -21,7 +26,27 @@ export const useAllowanceApproval = (
   const dispatch = useAppDispatch()
   const { showErrorToast } = useErrorHandler()
 
-  const sellAssetAccountId = useAppSelector(state => selectHopSellAccountId(state, hopIndex))
+  const secondHop = useAppSelector(selectSecondHop)
+  const isMultiHopTrade = useAppSelector(selectIsActiveQuoteMultiHop)
+  // TODO(gomes): this is temporary while devving - we should use the previous selectHopSellAccountId selector, if arity is happy there,
```

Comment:
The issue with `selectHopSellAccountId` is it isn't enforcing the arity of the newly added `selectSecondHopSellAccountId` with `chainId` and `accountNumber` params